pom.xml

```diff
@@ -1109,7 +1109,7 @@
       <dependency>
         <groupId>org.scala-lang.modules</groupId>
         <artifactId>scala-xml_${scala.binary.version}</artifactId>
-        <version>1.2.0</version>
+        <version>2.1.0</version>
```

Comment:
update this due to 

<img width="746" alt="image" src="https://user-images.githubusercontent.com/1475305/189395697-379c070a-737c-40af-ae7c-049623d86b67.png">